aws-scala-sdk/src/main/scala/com/itv/aws/package.scala

```diff
@@ -11,11 +11,10 @@ package object aws {
     Client
   ], Client](
       builder: AwsSyncClientBuilder[Builder, Client]
-  ): Regions => AWSCredentialsProvider => Client = {
-    (region: Regions) => credentials: AWSCredentialsProvider =>
-      {
-        builder.withRegion(region).withCredentials(credentials).build
-      }
+  ): Regions => AWSCredentialsProvider => Client = { (region: Regions) => credentials: AWSCredentialsProvider =>
+    {
+      builder.withRegion(region).withCredentials(credentials).build
+    }
```

Comment:
As a completely separate commit / PR - I think you should run scalafmt over the whole code base.